### PR TITLE
Change variable disable_tls to tls to avoid twice no like "if not dis…

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
Variable `disable_tls` was renamed to `tls` to avoid twice no like "if not disable_tls".
For CLI command it is possible to use the same old flag `--disable-tls` to disable mTLS.

### CODE REVIEW

File: openfl/interface/director.py [hunk 1]
```diff
@@ -31,15 +31,15 @@ def director(context):
 @director.command(name='start')
 @option('-c', '--director-config-path', default='director.yaml',
         help='The director config file path', type=ClickPath(exists=True))
-@option('--disable-tls', default=False,
-        is_flag=True)
+@option('--tls/--disable-tls', default=True,
```
Comment (psfoley) [inline openfl/interface/director.py:34]: This should just be `--tls`

File: openfl/interface/envoy.py [hunk 2]
```diff
@@ -33,8 +33,8 @@ def envoy(context):
         help='Current shard name')
 @option('-d', '--director-uri', required=True,
         help='The FQDN of the federation director')
-@option('--disable-tls', default=False,
-        is_flag=True)
+@option('--tls/--disable-tls', default=True,
```
Comment (psfoley) [inline openfl/interface/envoy.py:36]: Same comment as above. This should just be `@option('--tls', default=True,`

**Review by psfoley — CHANGES_REQUESTED** [review]
The changes are self explanatory. Once my two comments are addressed, I think this is ready for merge.

File: openfl/interface/director.py (hunk 1 above)
Comment (aleksandr-mokrov) [reply openfl/interface/director.py:34]: By default you can use `fx director start`  without TLS flag and variable `tls` will be set to `True`. But if you want to disable TLS, you can use `fx director start --disable-tls` and variable `tls` will be set to `False`. In other words, we disable mTLS only by explicitly passing the corresponding flag, by default mTLS is enabled.
[Click boolean flags specification](https://click.palletsprojects.com/en/8.0.x/options/#boolean-flags)

**Review by igor-davidyuk — APPROVED** [review]
seems legit

**Review by psfoley — APPROVED** [review]
Comments wer addressed. PR is approved.

**Pull request merged**